index.html

```diff
@@ -1,179 +1,250 @@
+            <section class="other">
+              <section class="informationCard">
+                <h3 class="subTitle">Multi-Post Stories Gaing+Glory</h3> 
+                <ul>
+                  <li class="myRecentWorks">Ruby on rails</li>
+                  <li class="myRecentWorks">css</li>
+                  <li class="myRecentWorks">JavaScript</li>
+                  <li class="myRecentWorks">html</li>
+                </ul>
+                <button type="button" class="theButton">See Project</button>
+              </section>
+            </section>
+          </div>  
+        </section>
+        <section class="about-me">
+          <img class="img1-aboutme" src="./imges/aboutme-img1.png">
+          <h2 class="color-blue">About me</h2>
+          <p class="color-blue">
+            I can help you build a product , feature or website Look through some 
+            of my work and experience! If you like what you see and have a project 
+            you need coded, don&apos;t hestiate to contact me.
+          </p>
+            <button type="button" class="btn-cv">Get My Resume</button>
+        </section>
+        <section class="skills-card">
+          <div class="img2-aboutme">
+            <section class="postOne">
+              <section class="my-placeholder">
+                <img src="./imges/Icon Name.svg" alt="skill"/>
+                <h3 class="subTitle">Languages</h3>
+                <ul>
+                  <li class="aboutMeListItem">Javascript</li>
+                  <li class="aboutMeListItem">Ruby</li>
+                  <li class="aboutMeListItem">Html</li>
+                  <li class="aboutMeListItem">CSS</li>
+                </ul>
+              </section>
+            </section>
+            <section class="postOne">
+              <section class="my-placeholder">
+                <img src="./imges/Group.svg" alt="skill"/>
+                <h3 class="subTitle">Frameworks</h3>
+                <ul>
+                  <li class="aboutMeListItem">Bootstrap</li>
+                  <li class="aboutMeListItem">Ruby on rails</li>
+                  <li class="aboutMeListItem">Rspec</li>
+                  <li class="aboutMeListItem">Capybara</li>
+                  <li class="aboutMeListItem">Selenium</li>
+                </ul>
+              </section>
+            </section>
+            <section class="postOne">
+              <section class="my-placeholder">
+                <img src="./imges/Icon Name.svg" alt="skill">
+                <h3 class="subTitle">Skills</h3>
+                <ul>    
+                  <li class="aboutMeListItem">Codekit</li>
+                  <li class="aboutMeListItem">Github</li>
+                  <li class="aboutMeListItem">Codepen</li>
+                  <li class="aboutMeListItem">Gitlab</li>  
+                  <li class="aboutMeListItem">Terminal</li>     
+                </ul>
+              </section> 
+            </section>
+          </div>
+        </section>  
```

Comment:
- [ ] [Optional] I am giving this optional as this is your last review please implement it. I would suggest you please decrease the width of the paragraph then only the background-image will render properly.
### Your design
![sssssssssssssssssssssssssssssss](https://user-images.githubusercontent.com/81727988/201500109-2b6b5440-2351-4bce-ab30-712cc3b8790d.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/201500112-a675b724-934b-472b-880c-46a8e221800f.png)